### PR TITLE
feat: Party management GUI enhancements

### PR DESCRIPTION
**Pull request by ryanzhoudev**
- PartyManager now correctly resets the party when retrieving the new party with /party list, instead of just appending new members to an invalid party
- Party management UI refreshes party status when opening with keybind
- New buttons!
    - "Refresh Party" force refreshes the party, in case anything goes wrong. Shouldn't really be necessary, but a nice to have
    - "Kick Offline", well, kicks offline players in the party. Only accessible when there are offline players & you are party leader
- Invite players has been moved, max characters extended, and a note added for bulk inviting
- Offline players now show up as ~Strikethrough~ on the player list
- Nearby players are now added to a suggestions list on the right. Nearby being a 30 block radius. (Pretty similar to Wynncraft's friend/party highlight range)
- Self now shows up as **Bold**
- Friends and nearby now have colours
- See image
- Fixes cascading lag (move static btns to init)
- Fixes static button sound being absurdly loud
image

Everything should be tested pretty well since I did the testing in a dxp weekend grind party, so tons of players in various states.
Also centered the entire UI properly. Was off by 5 pixels last time.

### CODE REVIEW

**Comment by magicus** [comment]
Not related to this PR, but a general wish since you're working on the party management gui: I typically want to team up with my friends. It would be nice if there were some way to have a list of common people to invite to a party. Either by parsing the friends list and showing all your friends (or, possibly, allow you to "star" a subset of your friends to have easy access to them if you have a lot of friends), or to just have a separate list of user names in the party management gui that you could add.

**Comment by ryanzhoudev** [comment]
Agreed, I'll also look in to a way to detect nearby players and add them to the suggestion list. Probably useful for grind parties

File: src/main/java/com/wynntils/modules/utilities/overlays/ui/PartyManagementUI.java [hunk 1]
```diff
@@ -233,7 +287,8 @@ private void updatePartyMemberList() {
         temporaryMembers.sort(sortPartyMembers);
         partyMembers.clear();
         partyMembers.addAll(temporaryMembers);
-        // partyMembers list is now sorted in [self, owner, party alphabetically]
+        partyMembers.addAll(offlineMembers);
```
Comment (magicus) [inline src/main/java/com/wynntils/modules/utilities/overlays/ui/PartyManagementUI.java:356]: Should offline members not be sorted alphabetically as well?

**Comment by magicus** [comment]
Yes, that'd be really nice. Not so simple to implement, though, I guess. Or maybe there is a "find entities within a BB" which you can use. (I have a vague memory about this). Also, I think we collect a list of your friends at login time. I'm not sure I remember correctly, nor if we actually use this for anything real, so if it is not being used it might have bit-rotted.

**Comment by ryanzhoudev** [comment]
Reviews appreciated, don't merge though, keeping the gui open slowly increases lag in-game until you close the gui so I'm trying to fix that

**Review by magicus — COMMENTED** [review]
I think this looks good.

If lag time is increasing, check that you are not accumulating stuff in some list.

**Comment by kristofbolyai** [comment]
Seeing the image, does this fit on different resolutions?

**Comment by ryanzhoudev** [comment]
> Seeing the image, does this fit on different resolutions?

Fits on 1020px width or larger. Imo this is fine as even 720p is 1280p wide. There's maybe another 70px maximum we could move the suggestions to the left. Legend fits great all the way down to 820px wide.

**Review by magicus — APPROVED** [review]
Nice! I've really been enjoying the party mgmt gui when playing on legacy. Keep up the good work.

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**